src/ui/buttons/props.js

```diff
@@ -430,7 +430,7 @@ export type ApplePaySessionConfigRequest = (
 
 export type ButtonMessage = {|
   amount?: number,
-  offer?: $ReadOnlyArray<$Values<typeof MESSAGE_OFFER>>,
+  offer?: $Values<typeof MESSAGE_OFFER> | string,
```

Comment:
Since the user input will be an array, should this be
``` js
offer?: $ReadOnlyArray<$Values<typeof MESSAGE_OFFER>> | string,
```

Reply:
that is output, offer in `ButtonMessageinput` is the array